.storybook/theme.scss

```diff
@@ -1 +1,102 @@
-@use '../scss.scss';
+@use '../scss.scss' with (
+	$global: (
+		'blue-0': #EBEFFF,
+		'blue-1': #B0C3F2,
+		'blue-2': #7AA2E5,
+		'blue-3': #4A89D8,
+		'blue-4': #1E78CB,
+		'blue-5': #1458A9,
+		'blue-6': #0C3D87,
+		'blue-7': #062665,
+		'blue-8': #021443,
+		'blue-9': #000721,
+
+		'red-0': #FF8A8A,
+		'red-1': #E65555,
+		'red-2': #CE2929,
+		'red-3': #B50505,
+		'red-4': #9C0000,
+		'red-5': #840000,
+		'red-6': #6B0000,
+		'red-7': #520000,
+		'red-8': #3A0000,
+		'red-9': #210000,
+
+		'orange-0': #FA714B,
+		'orange-1': #E24F23,
+		'orange-2': #C93302,
+		'orange-3': #B12E00,
+		'orange-4': #982A00,
+		'orange-5': #802500,
+		'orange-6': #682000,
+		'orange-7': #4F1900,
+		'orange-8': #371200,
+		'orange-9': #1F0B00,
+
+		'green-0': #80D971,
+		'green-1': #57C244,
+		'green-2': #34AB1F,
+		'green-3': #189402,
+		'green-4': #137D00,
+		'green-5': #0F6600,
+		'green-6': #0C4F00,
+		'green-7': #083800,
+		'green-8': #052100,
+		'green-9': #020A00,
+
+		'neutral-0': #E6EFFF,
+		'neutral-1': #C3CFE3,
+		'neutral-2': #A3B0C6,
+		'neutral-3': #8593AA,
+		'neutral-4': #69778E,
+		'neutral-5': #4F5C71,
+		'neutral-6': #384355,
+		'neutral-7': #232B39,
+		'neutral-8': #10151C,
+		'neutral-9': #000000,
+
+		'spacing-0': 0.25em, // 4px
+		'spacing-1': 0.5em, // 8px
+		'spacing-2': 1em, // 16px
```

Comment:
it's not always 16px .. might be deceptive?

Reply:
Correct - those comments are left over and will be removed - it was to help when adding all the values from figma into this file

Reply:
should the spacing be linear?
```
Array.from(new Array(10)).forEach((_, i) => console.log(`spacing-${i}: ${i * 0.25 + 0.25}em;`))
```
```
spacing-0: 0.25em;
spacing-1: 0.5em;
spacing-2: 0.75em;
spacing-3: 1em;
spacing-4: 1.25em;
spacing-5: 1.5em;
spacing-6: 1.75em;
spacing-7: 2em;
spacing-8: 2.25em;
spacing-9: 2.5em;
```